doc/choosing_a_combinator.md

```diff
@@ -26,7 +26,7 @@ Those are used to recognize the lowest level elements of your grammar, like, "he
 
 | combinator | usage | input | output | comment |
 |---|---|---|---|---|
-| [alt](https://docs.rs/nom/latest/nom/branch/fn.alt.html) | `alt(tag("ab") \| tag("cd"))` |  `"cdef"` | `Ok(("ef", "cd"))` |Try a list of parsers and return the result of the first successful one|
+| [alt](https://docs.rs/nom/latest/nom/branch/fn.alt.html) | `alt(tag("ab"), tag("cd"))` |  `"cdef"` | `Ok(("ef", "cd"))` |Try a list of parsers and return the result of the first successful one|
```

Comment:
```suggestion
| [alt](https://docs.rs/nom/latest/nom/branch/fn.alt.html) | `alt((tag("ab"), tag("cd")))` |  `"cdef"` | `Ok(("ef", "cd"))` |Try a list of parsers and return the result of the first successful one|
```

[`tag`](https://docs.rs/nom/7.0.0/nom/branch/fn.alt.html) takes a tuple of parsers.

Reply:
Thanks!